test/Microsoft.AspNet.Mvc.TagHelpers.Test/LinkTagHelperTest.cs

```diff
@@ -135,7 +282,56 @@ public void DoesNotRunWhenAllRequiredAttributesAreMissing()
             Assert.False(output.ContentSet);
         }
 
-        private TagHelperContext MakeTagHelperContext(
+        [Fact]
+        public void RendersLinkTagsForGlobbedHrefResults()
+        {
+            // Arrange
+            var context = MakeTagHelperContext(
+                attributes: new Dictionary<string, object>
+                {
+                    ["href"] = "/css/site.css",
+                    ["rel"] = "stylesheet",
+                    ["asp-href-include"] = "**/*.css"
+                });
+            var output = MakeTagHelperOutput("link", attributes: new Dictionary<string, string>
+            {
+                ["href"] = "/css/site.css",
+                ["rel"] = "stylesheet"
+            });
+            var logger = new Mock<ILogger<LinkTagHelper>>();
+            var hostingEnvironment = MakeHostingEnvironment();
+            var viewContext = MakeViewContext();
+            var globbingUrlBuilder = new Mock<IGlobbingUrlBuilder>();
+            globbingUrlBuilder.Setup(g => g.BuildUrlList("/css/site.css", "**/*.css", null))
+                .Returns(new[] { "/css/site.css", "/base.css" });
+            var helper = new LinkTagHelper
+            {
+                GlobbingUrlBuilder = globbingUrlBuilder.Object,
+                Logger = logger.Object,
+                HostingEnvironment = hostingEnvironment,
+                ViewContext = viewContext,
+                HrefInclude = "**/*.css"
+            };
+
+            // Act
+            helper.Process(context, output);
+
+            // Assert
+            Assert.Equal("<link href=\"/css/site.css\" rel=\"stylesheet\" />" +
+                         "<link href=\"/base.css\" rel=\"stylesheet\" />", output.Content);
+        }
+
```

Comment:
A functional test is fine with me, just some sort of output verification :smile: